readthedocs/templates/projects/project_import.html

```diff
@@ -1,82 +1,234 @@
-
-      {% if accounts.github.0 %}
-        <p>
-          {% blocktrans %}
-            Import one of your public GitHub repositories to start using your documentation.
-          {% endblocktrans %}
-        </p>
-
-        <form method="get" action="{% url "projects_import_github" %}">
-          <input type="submit" value="{% trans "Import from GitHub" %}"/>
-        </form>
-      {% else %}
-        <p>
-          {% blocktrans %}
-            Once your account is connected to your GitHub account,
-            you will be able to automatically import and configure your public GitHub repositories.
-          {% endblocktrans %}
-          <a href="{% provider_login_url "github" process="connect" next="/dashboard/import/" %}">Connect to GitHub</a>
-          {% trans "to import your first repository." %}
-        </p>
-      {% endif %}
-    </div>
 
-    <div class="col col-trisect">
-      <h3>From Bitbucket</h3>
-
-      {% if accounts.bitbucket.0 %}
-        <p>
-          {% blocktrans %}
-            Import one of your public Bitbucket repositories to start using your documentation.
-          {% endblocktrans %}
-        </p>
-
-        <form method="get" action="{% url "projects_import_bitbucket" %}">
-          <input type="submit" value="{% trans "Import from Bitbucket" %}"/>
-        </form>
-      {% else %}
-        <p>
-          {% blocktrans %}
-            Once your account is connected to your Bitbucket account,
-            you will be able to automatically import and configure your <b>public Bitbucket repositories</b>.
-          {% endblocktrans %}
-          <a href="{% provider_login_url "bitbucket" process="connect" next="/dashboard/import/" %}">Connect to Bitbucket</a>
-          {% trans "to import your first repository." %}
-        </p>
-      {% endif %}
+  <ul
+      class="notifications"
+      style="display: none;"
+      data-bind="visible: error">
+    <li
+        class="notification notification-40"
+        data-bind="with: error">
+
+      <span data-bind="visible: message">
+        {% trans "There was an error with your request:" %}
+        <span data-bind="text: message"></span>
+      </span>
+
+    </li>
+  </ul>
+
+  {% block main_content %}
+    <div class="col-major project-import-remote">
+
+      <button
+          data-bind="click: sync_projects, visible: !is_syncing()"
+          class="remote-sync">
+        <span>{% trans "Refresh your accounts" %}</span>
+      </button>
+      <span
+          data-bind="visible: is_syncing"
+          class="remote-sync-loader"
+          style="display: none;">
+        <img src="{% static 'core/img/loader.gif' %}"/>
+      </span>
+
+      <h1>{% trans "Import a Repository" %}</h1>
+
+      <div class="module repository-list">
+        <div class="module-wrapper">
+          <div class="module-list">
+            <div class="module-list-wrapper">
+
+              <ul data-bind="visible: is_ready() && !has_projects()" style="display: none;">
```

Comment:
I believe this is likely a side effect of the mentioned API filtering, since I had no connected accounts & had imported projects, which seems like an invalid state.